app/routes/queries/full-scan/FullScanForm.tsx

```diff
@@ -248,218 +259,256 @@ const FullScanForm = ({
+                                      'minimumStackSize',
+                                      ''
+                                    )
+                                    return
+                                  }
+                                  handleSearchParamChange(
+                                    'minimumStackSize',
 
-                          value.toString()
-                        )
-                      }}
-                      id="price_per_unit"
-                      className="flex-1 min-w-0 block w-full px-3 py-2 rounded-none rounded-l-md focus:ring-blue-500 focus:border-blue-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md"
-                    />
-                    <span
-                      className={`inline-flex items-center px-3 rounded-r-md border border-l-0 border-gray-300 bg-gray-50 text-gray-500 sm:text-sm`}>
-                      gil
-                    </span>
-                  </div>
-                  <p className="mt-2 text-sm text-gray-500">
-                    Desired Average Price Per Unit. ex: `10000` is only show
-                    deals that sell on average for 10000 gil or greater. For
-                    more items to sell choose a lower number.
-                  </p>
-                </div>
+                                    value.toString()
+                                  )
+                                }}
+                                id="minimum_stack_size"
+                                className="flex-1 min-w-0 block w-full px-3 py-2 focus:ring-blue-500 focus:border-blue-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md"
+                              />
+                            </div>
+                          </div>
 
-                <div className="col-span-6 sm:col-span-3 xl:col-span-2">
-                  <label
-                    htmlFor="filters"
-                    className="block text-sm font-medium text-gray-700">
-                    Item Filter
-                  </label>
-                  <div className={`mt-1 flex rounded-md shadow-sm`}>
-                    <input
-                      name="filters"
-                      value={ids.map((item) => item.toString())}
-                      hidden
-                      onChange={noop}
-                    />
-                    <button
-                      className="w-full py-2 px-4 text-sm bg-gray-100 border-gray-300 rounded text-left"
-                      aria-label="Choose filters"
-                      type="button"
-                      onClick={() => setIsOpen(true)}>
-                      Choose Filters
-                    </button>
-                  </div>
-                  <p className="mt-2 text-sm text-gray-500">
-                    Filters applied: {ids.length}
-                  </p>
-                </div>
+                          <div className="col-span-6 sm:col-span-3 xl:col-span-2">
```

Comment:
Can you make sure you run the linting to remove the white space please?

Reply:
I tried running the command but FullScanForm wasn't affected — I think the whitespace changes are caused by the addition of the Disclosure/Transition elements.
![image](https://user-images.githubusercontent.com/18511031/214377456-1169a58a-b3d6-4166-ba6f-3541eda4a742.png)

Reply:
Yeah just tried it in my editor; it isn't even showing this indentation, very crazy!
<img width="518" alt="Screenshot 2023-01-24 at 19 48 23" src="https://user-images.githubusercontent.com/51456734/214393305-a8bc1e5f-1cbc-45de-8559-435d1dde1117.png">